app/build.gradle

```diff
@@ -88,7 +94,10 @@ dependencies {
     implementation "androidx.compose.ui:ui-viewbinding:$rootProject.composeVersion"
     implementation "androidx.ui:ui-tooling:$rootProject.composeVersion"
     implementation "androidx.compose.runtime:runtime-livedata:$rootProject.composeVersion"
-    implementation "com.google.android.material:compose-theme-adapter:$rootProject.composeVersion"
+//    implementation "com.google.android.material:compose-theme-adapter:$rootProject.composeVersion"
```

Comment:
should this be removed rather than commented out?

Reply:
I'd keep it, we'll need to revert back to this for the actual release.

Reply:
+1, I left it commented out so that we don't forget to revert it